win32ss/user/ntuser/hook.c

```diff
@@ -1037,45 +1037,47 @@ BOOLEAN
 IntRemoveHook(PVOID Object)
 {
     INT HookId;
-    PTHREADINFO pti;
+    PTHREADINFO ptiHook;
     PDESKTOP pdo;
     PHOOK Hook = Object;
 
+
```

Comment:
Why do you add a blank line?

Reply:
To separate the block of variables from the block of assertions to be verified and the block of regular code...

Reply:
Yeah, I assume it belongs to the 2nd commit...